templates/generic/run_qunex.sh

```diff
@@ -17,13 +17,13 @@ scriptPath=$(dirname ${0})
 # -- key variables to set
 ParameterFolder='{{ QUNEX_PARAMETER_FILES }}'
 StudyFolder='{{ STUDY_FOLDER_SCRATCH }}'
-Subject='{{ SUBJECT_SESSION }}'
+Session='{{ SUBJECT_SESSION }}'
```

Comment:
Maybe just call the variable `SESSION`, rather than `SUBJECT_SESSION`?  But not sure if that might break other stuff...?

Reply:
To rename `SUBJECT_SESSION` --> `SESSION` we would need to rename everywhere. If we're going to do that, should we also change go ahead and change related variables:

https://github.com/humanconnectome/hcp-pipelines/blob/54a5e784ed1ea0013d368c71be0b3f8496b4fbe5/functions.py#L58-L63

Reply:
Also should SUBJECT_EXTRA, be "SCAN" ?

Reply:
Yes, it would be good to make these variable names more intuitive/appropriate.  I'm not actually sure what each is supposed to represent, so not sure what to suggest each variable name should be.  Could you make it concrete with an example what each of these represent?

Reply:
To launch the pipelines, the command is:
```$ ./run.sh StructuralPreprocessing CCF_HCA_STG:HCA0123456789:V1_MR:all```
The last argument is a colon delimited for `project:subject_id:classifier:extra`

Reply:
The extra is `all` for most runs, except in functional where it is the specific scan, I think. For example:
```./run.sh FunctionalPreprocessing CCF_HCA_TST:HCA6002236:V2_MR:rfMRI_REST1_AP```

Reply:
So, it seems like appropriate names for those lines in `functions.py` would be:
```
PROJECT
SUBJECT
CLASSIFIER
SCAN
_SCAN
SESSION
```

Reply:
Poking through the code though, it appears that in `functions.py` that `SUBJECT_EXTRA` is sometimes (for certain pipelines with HCD) expected to be "YOUNGER" or "OLDER".  In that case, it isn't always actually `SCAN`, and maybe it would be better to leave the variable as `EXTRA`?